capstone-ui/src/components/SearchPage/SearchPage.jsx

```diff
@@ -15,14 +15,13 @@ export default function SearchPage() {
   useEffect(() => {
     axios.get(`https://api-staging.codingcoach.io/mentors`).then((response) => {
       setMentors(response.data);
```

Comment:
If we change this to `setMentors(response.data.data);`
Then we don't need to refer to `mentor.data` in line 24. We can just do `mentor.filter`.
Would that work?